README.md

```diff
@@ -207,6 +207,25 @@ Optionally, you can add more command line arguments to the `command` section. Fo
   ```bash
   docker-compose -f /path/to/docker-compose.yml up -d
   ```
+## Running on Google Dataflow
+Copy the [Dataflow setup file](./deploy/dataflow/setup.py) over to where you wish to start your Dataflow jobs from.
+Replace the values of `SERVICE_NAME` and `GPROFILER_TOKEN`:
+  - Replace `<TOKEN>` in the command line with your token you got from the [gProfiler Performance Studio](https://profiler.granulate.io/installation) site
+  - Replace `<SERVICE NAME>` in the command line with the service name you wish to use
+whenever you start a Dataflow job, add the `--setup_file /path/to/setup.py` flag with your `setup.py` 
+ copy (**PLEASE NOTE** - the flag is `--setup_file` and not `--setup-file`).
+For example, here's a command that starts an example Apache Beam job with gProfiler:
+```shell
+python3 -m apache_beam.examples.complete.top_wikipedia_sessions \
+--region us-central1 \
+--runner DataflowRunner \
+--project my_project \
+--temp_location gs://my-cloud-storage-bucket/temp/ \
+--output gs://my-cloud-storage-bucket/output/ \
+--setup_file /path/to/setup.py
+```
+If you are already using the `--setup_file` flag for your own setup, please contact us or open a new GitHub issue [here](https://github.com/Granulate/gprofiler/issues/new).
```

Comment:
No simple way to explain how to extract the relevant parts from the sample `setup.py`?